Marlin/src/feature/fwretract.h

```diff
@@ -31,11 +31,12 @@
 
 class FWRetract {
 private:
+  
+
+public:
   #if EXTRUDERS > 1
     static bool retracted_swap[EXTRUDERS];         // Which extruders are swap-retracted
   #endif
-
-public:
   static bool autoretract_enabled,                 // M209 S - Autoretract switch
               retracted[EXTRUDERS];                // Which extruders are currently retracted
   static float retract_length,                     // M207 S - G10 Retract length
```

Comment:
To transmit status when swapping to next extruder , i need , to access to the old swap value , it's why this data become public